### PR TITLE
URL Updates

### PR DESCRIPTION
**Pull request by ellessenne**
This PR updates some URL (and language in the README file) after the repository transfer to the @openstatsware organisation.

### CODE REVIEW

File: _quarto.yml [hunk 1]
```diff
@@ -49,7 +49,7 @@ website:
     giscus:
       # Reference: https://quarto.org/docs/reference/projects/books.html#giscus
       # Also https://giscus.app/ for all options
-      repo: RConsortium/asa-biop-swe-wg
+      repo: openstatsware/website
       repo-id: "R_kgDOIEo72A"
```
Comment (danielinteractive) [inline _quarto.yml:53]: I guess these IDs can stay? Or do they also need to be updated?

**Review by danielinteractive — APPROVED** [review]
I guess if we are not sure we can merge this and see what happens 😄

File: _quarto.yml (hunk 1 above)
Comment (ellessenne) [reply _quarto.yml:53]: Good question! Where did you get the `repo-id` when you setup Giscus for the first time?

Comment (danielinteractive) [reply _quarto.yml:53]: On https://giscus.app/

Comment (ellessenne) [reply _quarto.yml:53]: Thanks – checked there and looks like it is all setup properly! 😃 (IDs are correct)

**Pull request merged**